speech_decoding/constants.py

```diff
@@ -0,0 +1,13 @@
+import torch, random
+import numpy as np
+
+# assert torch.cuda.is_available(), "Training without GPU is not supported."
+DEVICE = torch.device("cuda" if torch.cuda.is_available() else "cpu")
```

Comment:
This really shouldn't be hardcoded, unless for some reason it's not possible to run the code without a GPU (being too slow doesn't count). If you need to disable the GPU then that should be a flag.

Reply:
Understand. I've put back device and bar_format to how it was in `tidy`.